server/controllers/artistsController.js

```diff
@@ -43,6 +59,7 @@ const findOrCreateArtist = async (name, album) => {
 // }
 const getArtists = asyncHandler(async (req, res, next) => {
   const artistsArray = await findArtist(req);
+  logger.info(`getArtists with artistsArray ${artistsArray} created`);
```

Comment:
created? seems to me that the operation was to find, not to create

Reply:
that is right